### ISSUE TITLE
Bootstrap should utilize the method from the official documentation for remote state bucket management

### ISSUE THREAD

**Issue by kittydoor** [terraform-google-modules/terraform-example-foundation#1310]
### TL;DR

Bootstrapping includes a lot of manual steps. Improving backend.tf and GCS bucket management will improve it significantly!

### Detailed design

Current advice from the official docs (see [here](https://cloud.google.com/docs/terraform/resource-management/store-state)) includes a snippet to bootstrap the initial remote state bucket nicely (see code [here](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/storage/remote_terraform_backend_template/main.tf)).

This repo advises for [manual renaming and editing in README-GitHub.md](https://github.com/terraform-google-modules/terraform-example-foundation/blob/0a3e97714e6f5c3a6ebfb4704b69b0bffe7773a9/0-bootstrap/README-GitHub.md?plain=1#L235) of backend.tf.example.

Any reasons to not update to the proposed local_file approach?

### Additional information

Is there interest in developing this (relatively simple change) internally, or will a PR be considered for acceptance?

**Comment by lpezet** [comment]
This would be to replace the [cp mentioned in README-GitHub.md](https://github.com/terraform-google-modules/terraform-example-foundation/blob/0a3e97714e6f5c3a6ebfb4704b69b0bffe7773a9/0-bootstrap/README-GitHub.md?plain=1#L241) right?
The [`for...sed`](https://github.com/terraform-google-modules/terraform-example-foundation/blob/0a3e97714e6f5c3a6ebfb4704b69b0bffe7773a9/0-bootstrap/README-GitHub.md?plain=1#L244-L245) still need to happen or you have something else in mind?
